api/SpeechSynthesisEvent.json

```diff
@@ -201,37 +201,53 @@
           "spec_url": "https://wicg.github.io/speech-api/#dom-speechsynthesisevent-elapsedtime",
           "support": {
             "chrome": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "edge": {
-              "version_added": "14"
-            },
-            "firefox": {
-              "version_added": "49"
-            },
+              "version_added": "14",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
+            },
+            "firefox": [
+              {
+                "version_added": "95",
+                "notes": "Units are seconds (spec change). See <a href='https://bugzil.la/1732498'>bug 1732498</a>"
+              },
+              {
+              "version_added": "49",
+              "version_removed": "95",
+              "notes": "Units are miliseconds."
+              }
+            ],
             "firefox_android": {
-              "version_added": "62"
+              "version_added": "62",
+              "notes": "From version 95 the units are seconds. Earlier versions return miliseconds. See <a href='https://bugzil.la/1732498'>bug 1732498</a>"
```

Comment:
This is what I am calling _Option 1_ for recording the change. Essentially just a note that reflects the change version.

Option 2 is to highlight this more clearly by having a separate version - see: